spec/features/user_views_prs_spec.rb

```diff
@@ -116,9 +116,9 @@
       click_on "rails"
     end
 
-    expect(page).to have_content("An Ember PR")
-    expect(page).to have_content("A Rails PR")
-    expect(page).to have_content("A long pr")
+    expect(page).to have_content("An Ember PR", count: 1)
+    expect(page).to have_content("A Rails PR", count: 1)
+    expect(page).to have_content("A long pr", count: 1)
```

Comment:
Yes, though maybe I should only do this for the last PR assertion since that’s the only one with multiple tags…


On November 13, 2015 at 3:29:28 PM, Jason Draper (notifications@github.com) wrote:

In spec/features/user_views_prs_spec.rb:

> @@ -116,9 +116,9 @@
>        click_on "rails"
>      end
>   
> -    expect(page).to have_content("An Ember PR")
> -    expect(page).to have_content("A Rails PR")
> -    expect(page).to have_content("A long pr")
> +    expect(page).to have_content("An Ember PR", count: 1)
> +    expect(page).to have_content("A Rails PR", count: 1)
> +    expect(page).to have_content("A long pr", count: 1)
Does this test fail if you remove the fix?

—
Reply to this email directly or view it on GitHub.